badgecheck/util.py

```diff
@@ -6,6 +6,9 @@
 import requests_cache
 
 
+OPENBADGES_CONTEXT_URI_V2 = "https://w3id.org/openbadges/v2"
```

Comment:
Maybe, but I am wary of tying two things together that we may have reason later to not tie together. The software version of this validator is SemVer, meaning we have to make bumps to it based on certain conditions. At some point if we break the API contract, we might have to make a version change, and that would put it out of sync with the OB context version. 

We're actually going to go with a version of `(1,0,0,'dev1')` right now, so it's not even a clean start to a potential coupling. I don't know that util.py is the right place for this constant either though.